static/index.html

```diff
@@ -106,11 +107,6 @@ <h4>Other</h4>
         <ul>
           <li>say</li>
         </ul>
-
-        <h4>Internals</h4>
-        <ul>
-          <li>state <span class="explanation">returns a json-representation of the current state of player</span></li>
-        </ul>
```

Comment:
Every zone has a `/state` endpoint, so this should not be deleted.

Reply:
It is moved to a different area, and remains in the index.html as a clickable link. So /state is not deleted from the file.